ToDoApp/ToDoApp/ToDoListViewModel.swift

```diff
@@ -89,6 +86,7 @@ final class ToDoListViewModel: ObservableObject {
                 state = .failed(error)
             }
         }
+        loading = false
```

Comment:
Assigning false outside of the Task body will be executed immediately before the task is completed

Reply:
I have moved it inside Task, it's fixed now.